<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configure_Windows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PushButton_Save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Exit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>139</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerticalLayout_Key">
     <item>
      <widget class="QLabel" name="Label_Address_Key">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Port_Key">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_RunningMode_Key">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RunningMode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerticalLayout_Value">
     <item>
      <widget class="QLineEdit" name="Label_Address_Value">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBox_Port_Value">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>9600</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComboBox_RunningMode_Value">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>PlasticMemories</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SingleFile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MultiFile</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PushButton_Exit_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PushButton_Save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PushButton_Save_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Label_Address_Value</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>Label_Address_Value_TextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox_Port_Value</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SpinBox_Port_Value_ValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ComboBox_RunningMode_Value</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ComboBox_RunningMode_Value_CurrentIndexChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PushButton_Exit_Clicked()</slot>
  <slot>PushButton_Save_Clicked()</slot>
  <slot>Label_Address_Value_TextChanged()</slot>
  <slot>SpinBox_Port_Value_ValueChanged()</slot>
  <slot>ComboBox_RunningMode_Value_CurrentIndexChanged()</slot>
 </slots>
</ui>
